### PR TITLE
fixes problem solving text

### PR DESCRIPTION
**Pull request by esotrope**
@aaronsleeper @jessicacolton should we set a height for the titles so the claim views will line up?

screen shot 2017-07-28 at 5 11 20 pm

### CODE REVIEW

File: webapp/src/main/webapp/src/app/report/student-report-download.component.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export class StudentReportDownloadComponent extends ReportDownloadComponent {
         (error: any) => {
           // TODO: add handler for 404/500 errors - consider publishing to global notification component
           console.error(error);
+          this.subscription = null;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/report/student-report-download.component.ts]: unrelated change - the "completed" callback wasn't firing after the "error" callback

File: common/src/main/resources/messages.properties [hunk 2]
```diff
@@ -126,7 +126,7 @@ report.claim.4-CR.name=Research & Inquiry
 report.claim.4-CR.icon=fa-question
 report.claim.1.name=Concepts & Procedures
 report.claim.1.icon=fa-book
-report.claim.SOCK_2.name=Problem Solving
+report.claim.SOCK_2.name=Problem Solving & Modelling / Data Analysis
```
Comment (bgsavage) [inline common/src/main/resources/messages.properties]: Isn't modeling one `l`?

Comment (esotrope) [inline common/src/main/resources/messages.properties]: thanks - fixing

File: webapp/src/main/webapp/src/app/report/student-report-download.component.ts (hunk 1 above)
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/report/student-report-download.component.ts]: Hm, that's odd, I though the completed callback was supposed to be like a `finally` block.
